tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -1367,7 +1396,6 @@ def _impl(ctx):
             user_link_flags_feature,
             default_link_libs_feature,
             fdo_optimize_feature,
-            supports_dynamic_linker_feature,
```

Comment:
This was not supposed to be here and not in the default Xcode toolchain

Reply:
Probably too much effort, but should we land these changes first?